src/frontend/Canonicalize.ml

```diff
@@ -3,16 +3,25 @@ open Ast
 open Deprecation_analysis
 
 type canonicalizer_settings =
-  {deprecations: bool; parentheses: bool; braces: bool; inline_includes: bool}
+  { deprecations: bool
+  ; parentheses: bool
+  ; braces: bool
+  ; inline_includes: bool
+  ; strip_comments: bool }
 
 let all =
-  {deprecations= true; parentheses= true; inline_includes= true; braces= true}
+  { deprecations= true
+  ; parentheses= true
+  ; inline_includes= true
+  ; braces= true
+  ; strip_comments= false }
```

Comment:
It really should be called something like `legacy`, since it exists to be equivalent to what `--print-canonical` did before we allowed more fine-grained customization. Changing